### PR TITLE
Remove unused import json 

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY
The unused imports are removed by this PR so that the pylint check passes.

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: tests/sanity/ignore-2.15.txt [hunk 1]
```diff
@@ -8,3 +8,92 @@ plugins/modules/logs_metric_filter.py validate-modules:mutually_exclusive-type
...
+plugins/modules/eks_fargate_profile.py validate-modules:mutually_exclusive-type
+plugins/modules/redshift_endpoint_authorization.py validate-modules:mutually_exclusive-type
+plugins/modules/route53_key_signing_key.py validate-modules:mutually_exclusive-type
+plugins/modules/autoscaling_lifecycle_hook.py validate-modules:mutually_exclusive-type
+plugins/modules/ecs_primary_task_set.py validate-modules:mutually_exclusive-type
+plugins/modules/logs_metric_filter.py validate-modules:mutually_exclusive-type
+plugins/modules/wafv2_ip_set.py validate-modules:mutually_exclusive-type
+plugins/modules/wafv2_regex_pattern_set.py validate-modules:mutually_exclusive-type
+plugins/modules/wafv2_web_acl_association.py validate-modules:mutually_exclusive-type
+plugins/module_utils/core.py pylint!skip
```
Comment (alinabuzachis) [inline tests/sanity/ignore-2.15.txt:99]: I guess we can remove this one.

Comment (GomathiselviS) [reply tests/sanity/ignore-2.15.txt:99]: @alinabuzachis  Added this line to skip `ERROR: plugins/module_utils/core.py:46:0: unused-import: Unused ansible_dict_to_boto3_tag_list imported from utils`.

Comment (alinabuzachis) [reply tests/sanity/ignore-2.15.txt:89]: @GomathiselviS Something like this should be enough https://github.com/ansible-collections/amazon.aws/blob/main/plugins/module_utils/ec2.py#L46 It shouldn't be necessary to skip the whole file.

File: plugins/module_utils/core.py [hunk 2]
```diff
@@ -55,7 +55,7 @@
     json_patch,
     get_patch,
 )
-
+from .utils import ansible_dict_to_boto3_tag_list # pylint: disable=unused-import
```
Comment (alinabuzachis) [inline plugins/module_utils/core.py:58]: Please add also 
`from .utils import snake_dict_to_camel_dict # pylint: disable=unused-import`

And remove those imports from line 46.

**Review by alinabuzachis — COMMENTED** [review]
If I am not mistaken you should also add 

`from ansible.module_utils.common.dict_transformations  # pylint: disable=unused-import`

inside plugins/module_utils/utils.py.

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**